tools/hsettp/hsettp.c

```diff
@@ -144,12 +144,12 @@ handle_error(
     assert(cJSON_IsNumber(entry));
     err = cJSON_GetNumberValue(entry);
 
-    cJSON_Delete(body);
-
     fprintf(
         stderr, "%s: %s:%d: %s (%ld): %s (%d)\n", progname, file, lineno, title, status, detail,
         err);
 
+    cJSON_Delete(body);
+
```

Comment:
My bad, clearly I never should have trusted you ;-)

Reply:
First mistake was hiring me!